packages/api/src/server/analytics/activity-stream.entity.ts

```diff
@@ -36,5 +36,8 @@ export enum Action {
   ENV_SYNCED = 'ENV_SYNCED',
   WEBHOOK_CREATED = 'WEBHOOK_CREATED',
   WEBHOOK_DELETED = 'WEBHOOK_DELETED',
-  WEBHOOK_UPDATED = 'WEBHOOK_UPDATED'
+  WEBHOOK_UPDATED = 'WEBHOOK_UPDATED',
+  PERMISSION_CREATED = 'PERMISSION_CREATED',
```

Comment:
The access actions suppose to come from a database, then what's the purpose of this enum Action?

Reply:
This is for the Activity stream only, so we have already kept it in memory, because these are the constants. and  we'll fetch the Auth Actions details from the Database that is different.

Reply:
Since this is not a part of RBAC, Please create a dedicated PR for this and remove this from this PR.

Reply:
As discussed, I now understand the requirement, and it makes sense.